src/components/dashboard/overview/activities-card.tsx

```diff
@@ -86,57 +26,66 @@ const DonutCard = ({ title, className }: Props) => {
-                  <span
-                    className="absolute left-5 top-5 -ml-px h-full w-0.5 bg-gray-200"
-                    aria-hidden="true"
-                  />
-                ) : null}
-                <div className="relative flex items-start space-x-3">
-                  <>
-                    <div>
-                      <div className="relative px-1">
-                        <div className="flex h-8 w-8 items-center justify-center rounded-full bg-teal-100 ring-8 ring-white">
-                          <RiAccountCircleFill
-                            className="h-5 w-5 text-teal-500"
-                            aria-hidden="true"
-                          />
+        {activity.data.length ? (
+          <>
+            <ul role="list" className="-mb-8">
+              {activity.data.map((activityItem) => (
+                <li key={activityItem.id} className="group">
+                  <div className="relative pb-8">
+                    <span
+                      className="absolute left-5 top-5 -ml-px h-full w-0.5 bg-gray-200 group-last:hidden"
+                      aria-hidden="true"
+                    />
+
+                    <div className="relative flex items-start space-x-3">
+                      <>
+                        <div>
+                          <div className="relative px-1">
+                            <div className="flex h-8 w-8 items-center justify-center rounded-full bg-teal-100 ring-8 ring-white">
+                              <RiAccountCircleFill
+                                className="h-5 w-5 text-teal-500"
+                                aria-hidden="true"
+                              />
+                            </div>
+                          </div>
                         </div>
-                      </div>
-                    </div>
-                    <div className="min-w-0 flex-1 py-1.5">
-                      <div className="text-sm text-gray-500">
-                        <span className="font-medium text-primary/80">
-                          {activityItem.summary}
-                        </span>{" "}
-                        <br />
-                        <span className="whitespace-nowrap text-xs">
-                          {activityItem.date}
-                        </span>
-                      </div>
+                        <div className="min-w-0 flex-1 py-1.5">
+                          <div className="text-sm text-gray-500">
+                            <span className="font-medium text-primary/80">
+                              {activityItem.summary}
+                            </span>{" "}
+                            <br />
+                            <span className="whitespace-nowrap text-xs">
+                              {new Intl.DateTimeFormat("en-US", {
+                                dateStyle: "medium",
+                              }).format(activityItem.occurredAt)}
```

Comment:
Lets use time ago here, eg: 2 hours ago, 4 days ago etc. We could use a library for this, something like [javascript-time-ago](https://www.npmjs.com/package/javascript-time-ago)